development/model-comparison/app.py

```diff
@@ -86,7 +86,7 @@ def breast_cancer_sweep():
     h2 = MLPClassifier(9, 2)
 
     model_comparison = ModelComparison(h1, h2, training_set,
-                                       use_ml_flow=True, device="cuda")
+                                       device="cuda")
```

Comment:
Thanks. This also needs to be removed from line 192 - https://github.com/microsoft/BackwardCompatibilityML/blob/e62fac4704f3365b16323f6123f59f501654c04b/development/model-comparison/app.py#L192

Reply:
Fixed.